frontend/src/global_styles/content/_projects_list.sass

```diff
@@ -143,8 +143,11 @@ form.project-filters
   .generic-table--header-outer:hover
     background: initial
 
-  td.name a
-    white-space: nowrap
+  td.name
+    overflow: hidden
+    text-overflow: ellipsis
+    a
+      white-space: nowrap
```

Comment:
I think you should be able to use the `@include text-shortener` here that does all those three things